src/commands/ssh.ts

```diff
@@ -184,11 +184,11 @@ const ssh = async (args: yargs.ArgumentsCamelCase<SshCommandArgs>) => {
     print2("Did not receive access ID from server");
     return;
   }
-  const { id, isPreexisting } = response;
+  const { id, isPreexisting, event } = response;
   if (!isPreexisting) print2("Waiting for access to be provisioned");
 
   const requestData = await waitForProvisioning<AwsSsh>(authn, id);
-  const requestWithId = { ...requestData, id };
+  const requestWithId = { ...requestData, id, permission: event.permission };
```

Comment:
We rely on the returned permission object instead of waiting for the results of `waitForProvisioning` because SSH group requests workflow documents differ from SSH session request workflow documents. 

If we use the permission value from the SSH group request document we won't have a real instance arn to ssh into.